README.md

```diff
@@ -1060,6 +1065,93 @@ A targeted notification response will be returned as an XML representation, prov
 </targetedNotificationResponse>
 ```
 
+### Stored File
+
+A Stored File represents a file which is associated with a node which is stored in AWS.
+
+#### XML Representation
+
+##### Single Stored File
+
+``` xml
+<storedFile xmlns="http://schema.yudu.com" id="1234">
+  <fileName>video.mp4</fileName>
+  <fileType>MP4</fileType>
+  <fileSize>36259689</fileSize>
+  <supportedUsage>NOT_SPECIFIED</supportedUsage>
+  <nodeId>519</nodeId>
+  <links>
+    ⋮ // some link elements
```

Comment:
The links are a really key part of the response and it would be good to give users an idea of what to expect. Discoverability through links is a core feature of a good REST API.